sdk/cosmos/azure-cosmos-spark_3-0_2-12/pom.xml

```diff
@@ -25,6 +25,7 @@
     <maven.compiler.source>11</maven.compiler.source>
     <maven.compiler.target>11</maven.compiler.target>
     <jacoco.skip.coverage.check>true</jacoco.skip.coverage.check>
+    <shadingPrefix>azure_cosmos_spark</shadingPrefix>
```

Comment:
Should we have some versioning in the shading prefix (would it help to support multiple versions/deployments) - don't know just a question. Not blocking.